### DISCUSSION TITLE
Time format in wind input file

### DISCUSSION THREAD

**Discussion by fed1942** [NOAA-EMC/WW3#911] [Q&A]
I have the following problem: when running ww3_prep (indicating time in the wind input file) i get the error message "ERROR IN READING TIME FROM FILE (1)". Time format that i use for a block of data (i don't know what is wrong):
" 20230214 000000 "

When I do not indicate the time, ww3_prep does not throw an error. So, if I do not indicate the time, does ww3_prep recognize that I am entering data for more than one time step?

Thank you in advance.

**Comment by aliabdolali** [comment]
If you use time flap T or F, the behaviour would change.
see here https://github.com/NOAA-EMC/WW3/blob/develop/model/inp/ww3_prep.inp

**Reply by fed1942** [reply]
I am using time flap F (no time in the wind input file; data for two time steps), but i don't know if ww3_prep recognize more than 1 time step. I attach the prep.inp file.

![prep_input file](https://user-images.githubusercontent.com/78570105/220668154-be612c5e-09cc-4406-805b-96aff14522bb.png)
